init-tools.sh

```diff
@@ -19,6 +19,7 @@ case $OSName in
     Darwin)
         OS=OSX
         __DOTNET_PKG=dotnet-osx-x64
+        ulimit -n 2048
```

Comment:
Does that mean that if this script calls in to another sh script, that one won't have this setting? If so then we need to apply this in the init-tools in buildtools as well.